google/search-console/index.js

```diff
@@ -0,0 +1,98 @@
+import { fetchGSCProperties } from './src/siteList.js';
+import { getUrlsForProperty } from './src/urlsForProperty.js';
+import { inspectUrl } from './src/urlInspection.js';
+import { getTopKeywordsForUrl } from './src/keywords.js';
+
+const oauthToken = process.env.GOOGLE_SEARCH_OAUTH_TOKEN;
+
+async function main() {
+    const command = process.argv[2];
+    if (!oauthToken) {
+        console.error('Error: OAUTH_TOKEN environment variable is not set.');
```

Comment:
nit:

```suggestion
        console.error('Error: GOOGLE_SEARCH_OAUTH_TOKEN environment variable is not set.');
```

Reply:
Good catch! updated to use the variable set by the google cred.